src/libs/API.js

```diff
@@ -168,6 +168,10 @@ function handleExpiredAuthToken(originalCommand, originalParameters, originalTyp
  * @returns {Promise}
  */
 function request(command, parameters, type = 'post') {
+    // Always set referer to https://expensify.cash/
+    // eslint-disable-next-line no-param-reassign
+    parameters.referer = CONFIG.EXPENSIFY.URL_EXPENSIFY_CASH;
```

Comment:
Maybe it would be better to add this to `HttpUtils.xhr()`? It feels more correct to add lower in the chain and there is also another request where we might want to add this (although I am not sure)

https://github.com/Expensify/Expensify.cash/blob/20eed545afb55de9bf0fcb93592fe891de786cc9/src/libs/HttpUtils.js#L44-L47

https://github.com/Expensify/Expensify.cash/blob/b2c03fc5a8bd0d2c5a3f67847d132af1d41702e1/src/libs/Network.js#L36-L38

Reply:
I like adding it here rather than HttpUtils so that it doesn't create a dependency on `CONFIG`. (like maybe I want to use HttpUtils over in the comp tool). It also feels like this is a "business logic" type thing and not something that a lower-level lib would be concerned about. I think the lower-level lib is more concerned about the header for `referer` which we can't mess with anyway.

Reply:
Rather than disabling the lint rule, I suggest doing something like this:

```
const params = {...parameters, referer: CONFIG.EXPENSIFY.URL_EXPENSIFY_CASH};
```

Reply:
>so that it doesn't create a dependency on CONFIG. (like maybe I want to use HttpUtils over in the comp tool)

That's a good point. Mostly, I was thinking there might be multiple places where we might want to send the `referer` even though we aren't using `request()` (but I'm not sure if that's true or not)? If it is then it would make sense to set it once in one spot.

Alternative idea... add a method to add `n` params to every `HttpUtils.xhr()` request then you can use it in the comp tool or anywhere else to set a `referer` or just do nothing. Could be a private variable in the `HttpUtils` module.

Reply:
Another idea (if we don't care about skipping the `referer` outside of API calls) would be to enhance the parameters in the method we created to do that...

https://github.com/Expensify/Expensify.cash/blob/6913e20b724b33ba7885a97dc8ff0a845565b418/src/libs/API.js#L46-L70

This adds the `authToken` to every request before it calls `HttpUtils.xhr()`. So maybe we should change the name and also add the `referer` here?

Reply:
> So maybe we should change the name and also add the referer here?

Oh yes, perfect 💯 . I totally spaced out that method and agree totally. That's exactly what it was made for!

Reply:
🎯 - Nice find!